experimental/examples/prometheus/package.json

```diff
@@ -11,6 +11,6 @@
   "dependencies": {
     "@opentelemetry/api": "^1.0.2",
     "@opentelemetry/exporter-prometheus": "0.28.0",
-    "@opentelemetry/sdk-metrics-base": "0.28.0"
+    "@opentelemetry/sdk-metrics": "0.28.0"
```

Comment:
That example is somewhat outdated and `0.28.0` of the renamed package will not be published. I created an issue to update it and perform the rename later #3165.
I think we should leave `experimental/examples/prometheus` at `opentelemetry/sdk-metrics-base` for now.